src/Pluto.jl

```diff
@@ -94,6 +94,8 @@ export reset_notebook_environment
 export update_notebook_environment
 export activate_notebook_environment
 
+include("./precompile.jl")
```

Comment:
This file seems to be missing from the PR.

Reply:
Thanks Paul! Well spotted!